internal/cmd/kafka/command_consumer_list_onprem.go

```diff
@@ -0,0 +1,72 @@
+package kafka
+
+import (
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/examples"
+	"github.com/confluentinc/cli/internal/pkg/kafkarest"
+	"github.com/confluentinc/cli/internal/pkg/output"
+)
+
+func (c *consumerCommand) newListCommandOnPrem() *cobra.Command {
```

Comment:
Any way we can nicely combine the Cloud + On-Prem implementations in the same file?

Reply:
With the way that we currently initialize kafka clients for cloud and on-prem, I don't think so. Plus, `kafka-rest-sdk-go` doesn't have `Get` methods. So the kafka client initialization and the output code will both be in `if else` blocks.